solutions/usaco-714.mdx

```diff
@@ -238,3 +241,57 @@ int main() {
 	ps(ans);
 }
 ```
+</CPPSection>
+
+<JavaSection>
+```java
+import java.util.*;
+import java.io.*;
+
+public class WhyDidTheCowCrossTheRoad {
+	public static void main (String[] args) throws IOException {
+		Kattio io = new Kattio("helpcross");
+
+		int numChickens = io.nextInt();
+		int numCows = io.nextInt();
+
+		int[] chicken = new int[numChickens]; // where the chicken are located
+		int[] a = new int[numCows];
+		int[] b = new int[numCows];
+		int[] marked = new int[numCows];
+
+		for (int x = 0; x < numChickens; x++) {
+			chicken[x] = io.nextInt();
+		}
+		for (int x = 0; x < numCows; x++){
+			a[x] = io.nextInt();
+			b[x] = io.nextInt();
+		}
+		Arrays.sort(chicken);
+		int count = 0;
+
+		for (int x = 0; x < numChickens; x++) { // loop through each chicken
+			int smallest = -1;
+			// find the cow that ends the soonest that the chicken is in its range
+			for(int y = 0; y < numCows; y++){
```

Comment:
you put spaces between the for and the paren block up there
![image](https://user-images.githubusercontent.com/55369003/124368191-7deb6400-dc13-11eb-9a9a-f392dec53bfe.png)
why not do it here as well